march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -198,7 +190,7 @@ double InertiaEstimator::vibration_calculation()
   moa_ = mean(absolute(filtered_acceleration_array_));
   // aom = absolute of the mean
   aom_ = abs(mean(filtered_acceleration_array_));
```

Comment:
Instead of adding the comment, why not rename the variable to a more descriptive name like absolute_of_mean ?